assets/js/theme/account.js

```diff
@@ -27,7 +27,7 @@ export default class Account extends PageManager {
         this.passwordRequirements = this.context.passwordRequirements;
```

Comment:
![screen shot 2018-05-17 at 11 54 15 am](https://user-images.githubusercontent.com/5566572/40197748-3116137a-59c9-11e8-816e-04d2b9c85763.png)
think this is causing a problem for the CustomerAddressBookUITest

Reply:
![screen shot 2018-05-17 at 12 24 54 pm](https://user-images.githubusercontent.com/5566572/40199185-8af8f0a2-59cd-11e8-8a09-0cb4ce4f48de.png)
Functional\Tests\Customers\Stencil\CustomerSignInUITest

Reply:
tests/Functional/Tests/Customers/Stencil/CreateCustomerUITest.php

Reply:
tests/Functional/Tests/Customers/Stencil/RegisteredCustomerAccountSettingsUITest.php
tests/Functional/Tests/Customers/Stencil/RegisteredCustomerWishListUITest.php

Reply:
Fixed. Thanks for looking into this.